src/layout/DashboardLayout.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="h-screen flex flex-column">
+  <div class="h-screen flex flex-column align-items-center">
```

Comment:
Somehow, there is no margin between the bottom copyright banner and the content.